### PR TITLE
www: add newsletter.nodejs.org redirect

### PR DESCRIPTION
**Pull request by rvagg**
Fixes: https://github.com/nodejs/build/issues/648

### CODE REVIEW

File: setup/www/resources/config/nodejs.org [hunk 1]
```diff
@@ -184,6 +191,14 @@ server {
 }
 
 server {
+    listen *:443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name newsletter.nodejs.org;
```
Comment (jbergstroem) [inline setup/www/resources/config/nodejs.org]: this needs to include the ssl partial include

**Comment by mikeal** [comment]
I am +1 on this happening. However, I'm not familiar enough with the technical bits here to do a proper "review."

File: setup/www/resources/config/nodejs.org (hunk 1 above)
Comment (rvagg) [inline setup/www/resources/config/nodejs.org]: @jbergstroem can you explain this? I don't get it.

Comment (jbergstroem) [inline setup/www/resources/config/nodejs.org]: @rvagg: As always; I goofed. When I refactored how we add SSL certificates I put it in [a partial](https://github.com/nodejs/build/blob/master/setup/www/resources/config/ssl-defaults.conf), so my thought was that we had to include it. What I had forgotten was that we chose to include it as default and override it [where needed](https://github.com/nodejs/build/blame/master/setup/www/resources/config/iojs.org#L13..L15).

**Comment by rvagg** [comment]
@mikeal this is done now and is working, prefer the https version when handing it around or linking, sorry it's taken so long when it's actually very simple!

**Pull request closed without merging**